content/help/ops/networking/_index.md

```diff
@@ -4,3 +4,44 @@ description: Helps you manage the networking aspects of a running mesh.
+
+Another key concept to understand is how Istio performs traffic management.  This is described in [Traffic Management Explained](docs/concepts/#traffic-management).  Traffic management allows fine grained control with respect to what external traffic can enter or exit the mesh and how those requests are routed.   The traffic management configuration also dictates how requests between microservices within the mesh is handled.   Full details on how to configure the traffic management is available here: [Traffic Management Configuration](docs/tasks/traffic-management).
+
+TODO - Do we need to more completely describe gateways?  Not properly covered in concepts.
+## Network Layers Beneath the Mesh
+Istio's service mesh runs on top of the networking provided by the infrastructure environment on which the Istio mesh is deployed.  Istio has certain requirements of the networking layer.  This guide will not attempt to provide any operational insight to the networking layer as many options exist.  In the case of Kubernetes a good reference to understand the container networking layer [Kubernetes Cluster Operator](https://kubernetes.io/docs/user-journeys/users/cluster-operator/foundational/).  Istio has the following requirements of the networking infrastructure:
+* The mapping of a service name to workload IP is discoverable by Pilot (this is more a service discovery requirement than a networking requirement)
+* The Pilot discovery process can reach the environment specific API server for service discovery.
+* Service endpoints have L3 reachability to all endpoints for services in the Istio mesh.
+* Any firewall or ACL rules at the infrastructure environment networking layer don't conflict with any of the Istio layer 3-7 traffic management rules
+# Deployment Guidelines
+TODO - This would be about app deployment not Istio deployment,  but is this section needed?  What sort of topics to cover that are networking oriented.  Only obvious thing that comes to mind is appropriate namespace separation between control plane and applications.
+# Istio Control Plane
+When checking the network state of an Istio Deployment the first step should be to check on the operational status of the Istio control plane components.  It is best to start with Kubernetes commands.  Steps to enumerate:
+1. Kubectl get pods
+2. Kubectl get services
+3. Kubectl logs
+4. curl health
+5. curl status
+
+TODO - It might be possible to construct a check list of specific things to look for here. E.g. Specific log messages that provide good (or bad status),  but I think it's impossible for that ever to be complete enough and consistently will become out of date.
```

Comment:
One common problem is insufficient memory, where you need to look carefully for the error message:
```
    Last State:     Terminated
      Reason:       OOMKilled
      Exit Code:    137
      Started:      Wed, 25 Jul 2018 12:10:40 -0500
      Finished:     Wed, 25 Jul 2018 12:12:42 -0500
```
But this, and all this control plane op info, is not specifically networking related, so it would need to be at the top level in the ops guide, not here.